ims/common/config.py

```diff
@@ -73,5 +114,6 @@ def section(self, section_name, required=True):
 
 
 class ConfigSection:
+    """ A Simple Object to be used to get cfg.section.option """
     def __init__(self):
```

Comment:
This may not be related to current change, what does this ConfigSection do? - It seems its not having any methods.